nautobot/docs/user-guide/administration/installation/install_system.md

```diff
@@ -40,406 +40,444 @@ You must select either MySQL (MariaDB is **not** supported) or PostgreSQL. Postg
-    CREATE ROLE
-    postgres=# GRANT ALL PRIVILEGES ON DATABASE nautobot TO nautobot;
-    GRANT
-    postgres=# \connect nautobot
-    You are now connected to database "nautobot" as user "postgres".
-    nautobot=# GRANT CREATE ON SCHEMA public TO nautobot;
-    GRANT
-    nautobot=# \q
-    ```
+        ??? example "Entering Postgres DB"
 
-    #### Verify PostgreSQL Service Status
+            ```no-highlight
+            psql (12.5 (Ubuntu 12.5-0ubuntu0.20.04.1))
+            Type "help" for help.
 
-    You can verify that authentication works issuing the following command and providing the configured password. (Replace `localhost` with your database server if using a remote database.)
+            postgres=#
+            ```
 
-    If successful, you will enter a `nautobot` prompt. Type `\conninfo` to confirm your connection, or type `\q` to exit.
+        ```no-highlight
+        CREATE DATABASE nautobot;
+        CREATE USER nautobot WITH PASSWORD 'insecure_password';
+        GRANT ALL PRIVILEGES ON DATABASE nautobot TO nautobot;
+        \connect nautobot
+        GRANT CREATE ON SCHEMA public TO nautobot;
+        \q
+        ```
 
-    ```no-highlight
-    psql --username nautobot --password --host localhost nautobot
-    ```
+        ??? example "Example Postgres DB Creation Output"
+            ```no-highlight
+            postgres=# CREATE DATABASE nautobot;
+            CREATE DATABASE
+            postgres=# CREATE USER nautobot WITH PASSWORD 'insecure_password';
+            CREATE ROLE
+            postgres=# GRANT ALL PRIVILEGES ON DATABASE nautobot TO nautobot;
+            GRANT
+            postgres=# \connect nautobot
+            You are now connected to database "nautobot" as user "postgres".
+            nautobot=# GRANT CREATE ON SCHEMA public TO nautobot;
+            GRANT
+            nautobot=# \q
+            ```
 
-    Example output:
+        #### Verify PostgreSQL Service Status
 
-    ```no-highlight
-    Password for user nautobot:
-    psql (12.5 (Ubuntu 12.5-0ubuntu0.20.04.1))
-    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
-    Type "help" for help.
-
-    nautobot=> \conninfo
-    You are connected to database "nautobot" as user "nautobot" on host "localhost" (address "127.0.0.1") at port "5432".
-    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
-    nautobot=> \q
-    ```
+        You can verify that authentication works issuing the following command and providing the configured password. (Replace `localhost` with your database server if using a remote database.)
 
-=== "Ubuntu/Debian MySQL"
+        If successful, you will enter a `nautobot` prompt. Type `\conninfo` to confirm your connection, or type `\q` to exit.
 
-    ### Install MySQL
+        ```no-highlight
+        psql --username nautobot --password --host localhost nautobot
```

Comment:
How do you want to indicate that the user will be prompted for the password before continuing?

Reply:
I'm happy with having the example show the password. The copy and paste of the command gets things in place. If we add anything with password on the line of the no-highlight, we lose the proper copy/paste capability.